tests/tracing/conftest.py

```diff
@@ -92,3 +95,39 @@ def databricks_tracking_uri():
 def async_logging_enabled(request, monkeypatch):
     monkeypatch.setenv(MLFLOW_ENABLE_ASYNC_LOGGING.name, str(request.param))
     return request.param
+
+
+@pytest.fixture
+def otel_collector():
+    """Start an OpenTelemetry collector in a Docker container."""
+    subprocess.run(["docker", "pull", "otel/opentelemetry-collector-contrib"], check=True)
+
+    output_file = tempfile.NamedTemporaryFile().name
+
+    with open(output_file, "w") as f:
+        process = subprocess.Popen(
+            [
+                "docker",
+                "run",
+                "-p",
+                "127.0.0.1:4317:4317",
+                "otel/opentelemetry-collector",
+            ],
+            stdout=f,
+            stderr=subprocess.STDOUT,
+            text=True,
+        )
+
+    # Wait for the collector to start
+    time.sleep(5)
```

Comment:
just for my understanding, is this 5-second wait is the time where the collector is accepting requests, or is it actually just starting during this time?